src/App/Fossa/Config/ReleaseGroup/Create.hs

```diff
@@ -0,0 +1,114 @@
+  { releaseGroupCommon :: ReleaseGroupCommonOpts
+  , configOpts :: Maybe FilePath
+  , titleOpts :: Maybe Text
+  , releaseOpts :: Maybe Text
+  , projectsOpts :: Maybe [ReleaseGroupProjectOpts]
+  , licensePolicyOpts :: Maybe Text
+  , securityPolicyOpts :: Maybe Text
+  , qualityPolicyOpts :: Maybe Text
+  , teamsOpts :: Maybe [Text]
+  }
+  deriving (Eq, Ord, Show, Generic)
+
+cliParser :: Parser CreateOpts
+cliParser =
+  CreateOpts
+    <$> releaseGroupCommonOpts
+    <*> optional (strOption (applyFossaStyle <> long "config" <> short 'c' <> helpDoc configHelp))
+    <*> optional (strOption (applyFossaStyle <> long "title" <> short 't' <> stringToHelpDoc "The title of the FOSSA release group"))
+    <*> optional (strOption (applyFossaStyle <> long "release" <> short 'r' <> stringToHelpDoc "The release of the FOSSA release group"))
+    <*> optional (many (releaseGroupProjectOpts))
+    <*> optional (strOption (applyFossaStyle <> long "license-policy" <> short 'l' <> stringToHelpDoc "The name of the license compliance policy you want to assign to the FOSSA release group"))
+    <*> optional (strOption (applyFossaStyle <> long "security-policy" <> short 's' <> stringToHelpDoc "The name of the security policy you want to assign to the FOSSA release group"))
+    <*> optional (strOption (applyFossaStyle <> long "quality-policy" <> short 'q' <> stringToHelpDoc "The name of the quality policy you want to assign to the FOSSA release group"))
+    <*> optional (many (strOption (applyFossaStyle <> long "team" <> short 'T' <> stringToHelpDoc "The team you want to assign to the FOSSA release group")))
+
+mergeOpts ::
+  ( Has Diagnostics sig m
+  ) =>
+  Maybe ConfigFile ->
+  EnvVars ->
+  CreateOpts ->
+  m CreateConfig
+mergeOpts maybeConfig envVars cliOpts@CreateOpts{..} = do
+  apiOpts <- collectApiOpts maybeConfig envVars releaseGroupCommon
+  releaseGroupRevision <- collectReleaseGroupRevision maybeConfig cliOpts
+  pure $ CreateConfig apiOpts releaseGroupRevision
+
+collectReleaseGroupRevision :: (Has Diagnostics sig m) => Maybe ConfigFile -> CreateOpts -> m ReleaseGroupRevision
+collectReleaseGroupRevision maybeConfig CreateOpts{..} = do
+  let licensePolicy = licensePolicyOpts <|> (maybeConfig >>= configReleaseGroup >>= configReleaseGroupLicensePolicy)
+      securityPolicy = securityPolicyOpts <|> (maybeConfig >>= configReleaseGroup >>= configReleaseGroupSecurityPolicy)
+      qualityPolicy = qualityPolicyOpts <|> (maybeConfig >>= configReleaseGroup >>= configReleaseGroupQualityPolicy)
+      -- NOTE: teamsOpts and projectsOpts default to Just [] when it is not set through CLI flags.
+      --       Convert these to Nothing so we can try to extract from the config file.
+      teams = case teamsOpts of
+        Nothing -> (maybeConfig >>= configReleaseGroup >>= configReleaseGroupTeams)
+        Just [] -> (maybeConfig >>= configReleaseGroup >>= configReleaseGroupTeams)
+        Just teamsOpts' -> Just teamsOpts'
```

Comment:
I have changed our parser to use [some](https://hackage.haskell.org/package/base-4.16.3.0/docs/Control-Applicative.html#v:some) instead of `many`. This won't be an issue anymore.